src/Runtime/python/onnxmlirdocker.py

```diff
@@ -58,6 +58,7 @@ class InferenceSession:
     def __init__(self, model_path, **kwargs):
         self.debug = False
         self.session = None
+        self.output_dir = tempfile.TemporaryDirectory()
```

Comment:
We want to keep this temp folder, so move it to the class member. Otherwise, the temp folder is cleaned up automatically.